compile_upload.R

```diff
@@ -1,8 +1,5 @@
 ## this script is for pre-compiling data and running/uploading app
 
-# clean slate
-rm(list=ls())
```

Comment:
Hi Sharla! Just curious why you removed this. I used it to clean up the environment before running all the scripts further below and saving the objects in memory into pre-compiled.RData -- it was an effort to make the whole thing more reproducible. Maybe you're doing this in another way or what I did was unnecessary -- just let me knwo!

Reply:
Here's a good article on the topic, specifically the section "What’s wrong with rm(list = ls())?
": https://www.tidyverse.org/articles/2017/12/workflow-vs-script/

I prescribe to this idea of workflows -- essentially, rm(list=ls()) doesn't ensure that you have a clean session, and the best way to ensure this is by actually restarting R. Everything that is required should be in `compile_upload.R`, so if we assume R is restarted before it's run, then there's no need to remove objects from the workspace.

Reply:
k makes sense! Too bad there isn't an easy way to automate this. I just like to make things foolproof when I can, rather than depending on myself to restart an R session. E.g. these don't seem like quite what we would want in place of `rm(list=ls()`: https://stackoverflow.com/questions/6313079/quit-and-restart-a-clean-r-session-from-within-r